tests/kernel/run_ltp.pm

```diff
@@ -259,6 +259,8 @@ sub run {
 
     assert_script_run('cd /opt/ltp/testcases/bin');
 
+    script_run('ver_linux');
```

Comment:
@richiejp No, as I added `LTPROOT` env. variable (aka `/opt/ltp`) into `PATH` env. variable in this commit. But I'll change it to use full path, but with use of `LTPROOT` instead adding new entry into `PATH`.

BTW: It would be nice to use `$LTPROOT/testcases/bin` instead of `/opt/ltp/testcases/bin` (one day we might leave `/opt`.